Sources/BraveWallet/Extensions/RpcServiceExtensions.swift

```diff
@@ -296,7 +296,8 @@ extension BraveWalletJsonRpcService {
     }
     if let data = metaDataString.data(using: .utf8),
        let result = try? JSONDecoder().decode(NFTMetadata.self, from: data) {
-      return result
+      let r =  result.httpfyIpfsUrl(ipfsApi: ipfsApi)
+      return r
```

Comment:
nit: you can just return directly here:
```suggestion
      return result.httpfyIpfsUrl(ipfsApi: ipfsApi)
```